M2/Macaulay2/m2/local.m2

```diff
@@ -4,8 +4,8 @@
 
 truncatedDual := (M,e) -> (
      -- find (k-dual M), truncated in degrees >= e.
+     -- depends on truncate methods
      needsPackage "Truncations";
```

Comment:
This call to `needsPackage` is redundant, because Truncations is a pre-loaded package.

And it's a hack.

It would be good to move this code into a package that explicitly imports "Truncations".

Reply:
Sorry if I wasn't clear, but this PR removes Truncations from the preloaded list in order to increase speed.

Reply:
See #1377.